### PR TITLE
Remove grpc tool related code from gRPC repo 

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/3828
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: ballerina/build.gradle [hunk 1]
```diff
@@ -219,6 +219,29 @@ publishing {
...
+                println stdOut.toString()
+                sleep(5 * 1000)
+            } else {
+                println "LDAP server is already running."
+            }
+        }
+    }
+}
+
 build.outputs.dir artifactLibParent // To pack Ballerina gRPC CLI
```
Comment (dilanSachi) [inline ballerina/build.gradle:245]: we need to remove this right?

**Review by daneshk — APPROVED** [review]

File: ballerina/build.gradle (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina/build.gradle]: Removed in [c3bb8f0](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1122/commits/c3bb8f086c6c26131311d2ad823467f3bfcb4275)

File: .github/workflows/build-with-bal-test-native.yml [hunk 2]
```diff
@@ -29,4 +29,4 @@ jobs:
             native_image_options: ${{ inputs.native_image_options }}
             additional_ubuntu_build_flags: '-x :grpc-native:test -x :grpc-compiler-plugin-tests:test -x :grpc-tooling-tests:test'
             additional_windows_build_flags: '-x :grpc-native:test -x :grpc-compiler-plugin-tests:test -x :grpc-tooling-tests:test'
-            java_tool_options: '-Dfile.encoding=UTF8'
+            java_tool_options: '-Dfile.encoding=UTF8'
```
Comment (dilanSachi) [inline .github/workflows/build-with-bal-test-native.yml:32]: Shall we add a newline here as well. I must have missed this when adding tests

**Review by dilanSachi — APPROVED** [review]
Added a minor comment. Others LGTM

File: .github/workflows/build-with-bal-test-native.yml (hunk 2 above)
Comment (MadhukaHarith92) [reply .github/workflows/build-with-bal-test-native.yml]: Added in [d3cab60](https://github.com/ballerina-platform/module-ballerina-grpc/pull/1122/commits/d3cab606191ea7c4c705a00a36d6439632781502)

**Pull request merged**